### DISCUSSION TITLE
Fix the OutRun MODE Scoring

### DISCUSSION THREAD

**Discussion by Sonic-TV** [emoose/OutRun2006Tweaks#232] [Suggestions]
### Describe the feature you'd like to see added

Make the smaller cars score 1500 rather than 750 pts to match the arcade scoring.
i.e. if the car is not a convoy or a bus or a rival cars, they should all score 1500pts when non slipstream (3000 pts if slipstream).

**Comment by Sonic-TV** [comment]
A seperate leaderboard may required for this fix (as it's hard to mod the console versions).

**Comment by Dekaid0011** [comment]
This recently came back up in the Discord, scoring is fixed only in the PAL psp release. It's also able to connect to the leaderboards and post scores, so could be an "unfair" advantage?

Its ofc more difficult to get good scores on psp, but I was already able to get 50k more points than my best attempt of 15 Cont. 2SP on pc with a much more messy run on psp.

**Reply by Sonic-TV** [reply]
> This recently came back up in the Discord, scoring is fixed only in the PAL psp release. It's also able to connect to the leaderboards and post scores, so could be an "unfair" advantage?
> 
> Its ofc more difficult to get good scores on psp, but I was already able to get 50k more points than my best attempt of 15 Cont. 2SP on pc with a much more messy run on psp.

Right. Just unify the scoring as the arcade's.

**Comment by KawaiiFiveO** [comment]
I've created a fork that implements this feature. (RestoreArcadeScoring)
https://github.com/KawaiiFiveO/OutRun2006Tweaks

Enabling the patch will also disable online functionality.

**Reply by Sonic-TV** [reply]
You probably wanna merge this into the main repo. Maybe a separate leaderboard is possible.
